src/components/Reusable/Input.js

```diff
@@ -0,0 +1,13 @@
+import { useTheme } from "../../context/theme-context"
+import { getBgColor, getTextColor } from "../../utils"
+
+const Input = (props) => {
+    const { theme } = useTheme()
+
+    return (
+        <input type={props.type} placeholder={props.placeholder} value={props.value} onChange={props.onChange}
+            className={`${getBgColor(theme)} ${getTextColor(theme)} ${props.classes}`} />
+    )
```

Comment:
Here too you can destructure props object.

Reply:
i have made the changes. thanks